ui/app/AppLayouts/Browser/stores/WalletStore.qml

```diff
@@ -17,6 +17,7 @@ QtObject {
 
     function switchAccountByAddress(address) {
         browserSectionCurrentAccount.switchAccountByAddress(address)
+        walletSectionTransactions.setModel(address)
```

Comment:
can can this not be handled in BE itself? Feels weird that the UI need to call this from here. UI should simply send request to switch account and everything else should be handled in BE?

Reply:
I hesitated! I need to move all the tx stuff in current account itself. do u think it is worth it?

Reply:
I really think we should, leaving this to UI means it could lead to an error in case UI doesnt call required functions. also much cleaner with a single switchAccount call no? wdyt

Reply:
alright will do that change then

Reply:
Just realized that the getWallet API will also deal with this